src/events/2018-03-14.md

```diff
@@ -33,11 +33,13 @@ agenda:
     description: Chat, mingle and soak in all you've learned so far
   - time: 8:30pm
     type: speaker
-    title: "[Ruth John](https://twitter.com/rumyra) - Title TBC"
+    title: "[Ruth John](https://twitter.com/rumyra) - Web Workers"
     description: |
       Ruth will talk about web workers.
 
-      Full talk description and bio TBC.
+      ---
+
+      _Ruth is a creative engineer with a web development background. She has enjoyed a fifteen year career working on websites, applications, and most recently interactive art projects, especially those featuring audio. She also educates people. As a Google Developer Expert and founding member of [{ Live : JS }](https://livejs.network/) she&#8217;s almost always got a conference talk lined up, article ready to be published, or live show tour date in the diary._
```

Comment:
I made some small edits to Ruth's bio.

* Hyperlinked to `{ Live : JS }`
* I added [Oxford commas](https://en.oxforddictionaries.com/explore/what-is-the-oxford-comma).